typings/css-properties.d.ts

```diff
@@ -1926,7 +1926,7 @@ export interface CSSPropertiesLossy {
     | undefined
     | Array<CSSPropertiesComplete[keyof CSSPropertiesComplete]>
     | CSSPropertiesLossy
-    | React.ReactChild
+    | React.ReactNode
```

Comment:
Thanks for the update, I'd love to remove this entirely from this file as a ReactNode/ReactChild is not a css property.

But keeping things working in the meantime sounds great 💯.

Reply:
Me too, but small steps at a time. :-(

I have tried investigating the root cause, but I can not understand why does `children` match [the index signature on `CSSProperties`](https://github.com/denis-sokolov/glamorous/blob/ee948b134851ff9375a18a866fccc3c81d968488/typings/css-properties.d.ts#L1922) rather than the existing `children` on React components. I tried overriding `children` even directly in `CSSPropertiesLossy`, but the index still takes priority. By the way, if I remove `CSSPropertiesLossy` entirely, my code typechecks. But it, of course, might not be the solution.